modelsim_run/work/_info

```diff
@@ -37,7 +37,7 @@ Z12 tCvgOpt 0
 vrvc_asap_5pl
 R1
 Z13 DXx4 work 12 rvc_asap_pkg 0 22 8`Y[7m:YUj8ZFKKf]kea>2
-Z14 !s110 1653588995
+Z14 !s110 1653833359
```

Comment:
Lest add this to the git ignore.
First remove the file and commit, then add it to the ignore list

Reply:
Taken care of.